src/applications/bmqbrkr/bmqbrkr.m.cpp

```diff
@@ -672,6 +651,46 @@ int main(int argc, const char* argv[])
         return mqbu::ExitCode::e_COMMAND_LINE;  // RETURN
     }
 
+    if (version) {
+        bsl::cout << bmqbrkrscm::Version::version() << "\n";
+        return 0;
+    }
+
+    if (configDir.empty()) {
+        bsl::cerr << "Error: No value supplied for the non-option argument "
+                     "\"config\".\n";
```

Comment:
Broker should exit if no cfg dir is specified, right?

Reply:
Ah yes.